lightning/restore/restore.go

```diff
@@ -490,12 +490,17 @@ func (rc *RestoreController) restoreTables(ctx context.Context) error {
 	}
 	taskCh := make(chan task, rc.cfg.App.IndexConcurrency)
 	defer close(taskCh)
+	oriGCLifeTime, err := ObtainGCLifeTime(ctx, rc.tidbMgr.db)
+	if err != nil {
+		return errors.Trace(err)
```

Comment:
```suggestion
		return err
```

Let's not use unnecessary `errors.Trace` in new code 🙂